olp-cpp-sdk-dataservice-read/src/repositories/DataRepository.cpp

```diff
@@ -162,10 +162,25 @@ BlobApi::DataResponse DataRepository::GetBlobData(
     }
   }
 
+  // Check if other threads have faced an error.
+  const auto optional_error = mutex.get_error();
+  if (optional_error) {
+    OLP_SDK_LOG_DEBUG_F(kLogTag,
+                        "This blob download ended up with an error in other "
+                        "thread, hrn='%s', key='%s', error='%s'",
```

Comment:
Bit of a mouthful for a log message

Reply:
That is why this is debug log and not warning :)